### ISSUE TITLE
NTR: Vaginal mass  (Endocrine & genitourinary)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7960]
New term request Vaginal mass  (elthat_150214030019):   UMLS: 0221783

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000105
- label: Vaginal mass
- parents: [Abnormal vagina morphology (HP:0000142)](https://hpo.jax.org/app/browse/term/HP:0000142)
- pmid: 
- def: A swelling or enlargment localized to the vagina. The word mass is usually used at an early stage of the diagnostic workup before the precise nature of the swelling has been determined.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7960
- synonyms:
